src/mapeo-project.js

```diff
@@ -840,6 +849,35 @@ async function deleteAll(dataType) {
   return Promise.all(deletions)
 }
 
+/**
+@param {Object} dataTypes
+@param {MapeoProject['translation']} dataTypes.translation
+@param {MapeoProject['preset']} dataTypes.presets
+@param {MapeoProject['field']} dataTypes.fields
+*/
+async function deleteTranslations(dataTypes) {
+  const deletions = []
+  for (const {
+    docId,
+    docIdRef,
+    schemaNameRef,
+  } of await dataTypes.translation.getMany()) {
+    try {
+      if (schemaNameRef === 'presets' || schemaNameRef === 'fields') {
+        const toDelete = await dataTypes[schemaNameRef].getByDocId(docIdRef)
+        if (toDelete.deleted)
+          deletions.push(dataTypes.translation.delete(docId))
+      }
+    } catch (e) {
+      console.log(
```

Comment:
Nit: should this use `this.#l` instead of `console.log`?

Reply:
yeah, this makes sense. But the function is outside of the class. I wrapped it in a closure (not ideal but I think it makes sense for this fn to live outside the class